### ISSUE TITLE
Vulnerable components delivered in the 'open-liberty' Docker container.

### ISSUE THREAD

**Issue by mccright** [OpenLiberty/open-liberty#10950]
I was unable to find a dedicated channel for  
reporting vulnerabilities and noticed that some  
vulnerabilities have been reported as issues (for example:  
https://github.com/OpenLiberty/open-liberty/issues/4409),   
so I am reporting here.  

Thank you for letting us know -

We use Hackerone to manage vulnerability disclosure. To disclose a vulnerability in Open Liberty please use https://hackerone.com/ibm to enter the details. It will be routed to the Open Liberty project so a fix can be provided prior to public disclosure.

The policy on the https://hackerone.com/ibm describes the policy guidelines for IBM products. Open Liberty is an IBM open source project so some of the guidelines do not apply. Specifically the first and third bullets can be ignored since they only apply to IBM products.

**Comment by arthurdm** [comment]
Thanks for letting us know @mccright 

That particular tag of Open Liberty is built upon the Ubuntu 18.04 image (indirectly via the AdoptOpenJDK OpenJ9 image), which we noticed has not been updated in about a month.  I have now opened a private bug (since it has to do with vulnerabilities) in their repository referencing these CVEs.  Once they fix these our images will be automatically rebuilt by Docker Inc.
